nats-server/configuration/logging.md

```diff
@@ -64,12 +64,15 @@ All of these settings are available in the configuration file as well.
 debug:   false
 trace:   true
 logtime: false
+log_size_limit: 1GB
```

Comment:
Since I suggest we don't document the command line parameter (which is -log_size_limit), I would use here the `logfile_size_limit` name version which hints that it applies to logs that are of "file" nature.

Reply:
changed